commerce_coordinator/apps/lms/views.py

```diff
@@ -247,22 +273,31 @@ def post(self, request) -> Response:
                             'attribute.', status=HTTP_400_BAD_REQUEST)
 
         if not order_line_item_id:
-            logger.error(f"[RefundView] Failed processing refund for order {order_id} for username: {username}, "
-                         f"course_id: {course_id} the enrollment_attributes array requires an orders: line_item_id "
-                         f"attribute.")
-            return Response('the enrollment_attributes array requires an orders: line_item_id '
-                            'attribute.', status=HTTP_400_BAD_REQUEST)
+            log_msg = (
+                f"[RefundView] Failed processing refund for order {order_id} for username: {username}, "
+                f"entitlement_id: {entitlement_id}. Line item does not exist."
+            ) if entitlement_refund else (
+                f"[RefundView] Failed processing refund for order {order_id} for username: {username}, "
+                f"course_id: {course_id} the enrollment_attributes array requires an orders: line_item_id "
+                f"attribute."
+            )
+            logger.error(log_msg)
+
+            response_msg = 'Line item does not exist.' if entitlement_refund else \
+                'the enrollment_attributes array requires an orders: line_item_id attribute.'
+            return Response(response_msg, status=HTTP_400_BAD_REQUEST)
 
         try:
             result = OrderRefundRequested.run_filter(order_id, order_line_item_id)
 
+            log_type = f"{entitlement_id=}" if entitlement_refund else f"{course_id=}"
```

Comment:
```suggestion
            info_log_details = f"{entitlement_id=}" if entitlement_refund else f"{course_id=}"
```

Reply:
its log type, not log detail

Reply:
umm, i thought we are just storing what data to print in log in `log_type` and log type refers to types such as `warn` `debug` and `info` logs.